x/emissions/migrations/v2/migrations.go

```diff
@@ -3,16 +3,27 @@ package v2
 import (
 	"fmt"
 
+	"cosmossdk.io/collections"
+	emissionsTypes "github.com/allora-network/allora-chain/x/emissions/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// An example MigrateStore function that might be run in the migration
-// handler.
-func MigrateStore(ctx sdk.Context) error {
-	ctx.Logger().Error(fmt.Sprintf(
+// for upgrade v0.2.8, delete the contents of
+// the stakeRemoval and delegateStakeRemoval
+// key value stores
+func MigrateStore(
+	ctx sdk.Context,
+	stakeRemoval *collections.Map[collections.Pair[uint64, string], emissionsTypes.StakePlacement],
+	delegateStakeRemoval *collections.Map[collections.Triple[uint64, string, string], emissionsTypes.DelegateStakePlacement],
+) error {
+	ctx.Logger().Warn(fmt.Sprintf(
 		"###################################################" +
 			"### MIGRATING STORE FROM VERSION 1 TO VERSION 2 ###" +
```

Comment:
no semver here?

Reply:
doesn't appear that's how the modules themselves track versioning, i agree it's weird but it's in line with how other chains do it

https://github.com/evmos/evmos/blob/49cab07c39f64be0b260d67423c12380a475ce4f/x/evm/module.go#L28-L29

https://github.com/osmosis-labs/osmosis/blob/5b92bcb75ca8f9caae83a2a928d2426afa822f82/x/gamm/module.go#L155-L156